[2025-09-03T10:04:01.992+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-03T10:04:02.018+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: market_table_foreign.foreign_HNX scheduled__2025-09-03T10:03:00+00:00 [queued]>
[2025-09-03T10:04:02.028+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: market_table_foreign.foreign_HNX scheduled__2025-09-03T10:03:00+00:00 [queued]>
[2025-09-03T10:04:02.028+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 11
[2025-09-03T10:04:02.048+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): foreign_HNX> on 2025-09-03 10:03:00+00:00
[2025-09-03T10:04:02.058+0000] {standard_task_runner.py:63} INFO - Started process 2727536 to run task
[2025-09-03T10:04:02.062+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'market_table_foreign', 'foreign_HNX', 'scheduled__2025-09-03T10:03:00+00:00', '--job-id', '70543', '--raw', '--subdir', 'DAGS_FOLDER/dag_market_live.py', '--cfg-path', '/tmp/tmpea61i49i']
[2025-09-03T10:04:02.065+0000] {standard_task_runner.py:91} INFO - Job 70543: Subtask foreign_HNX
[2025-09-03T10:04:02.117+0000] {task_command.py:426} INFO - Running <TaskInstance: market_table_foreign.foreign_HNX scheduled__2025-09-03T10:03:00+00:00 [running]> on host 1877907bc043
[2025-09-03T10:04:02.276+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='market_table_foreign' AIRFLOW_CTX_TASK_ID='foreign_HNX' AIRFLOW_CTX_EXECUTION_DATE='2025-09-03T10:03:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-03T10:03:00+00:00'
[2025-09-03T10:04:02.277+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-03T10:04:02.292+0000] {foreign_HNX.py:17} INFO - Kết nối DB
[2025-09-03T10:04:02.614+0000] {foreign_HNX.py:42} INFO - Đã lưu foreign_HNX
[2025-09-03T10:04:02.616+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-09-03T10:04:02.617+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-03T10:04:02.631+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=market_table_foreign, task_id=foreign_HNX, run_id=scheduled__2025-09-03T10:03:00+00:00, execution_date=20250903T100300, start_date=20250903T100402, end_date=20250903T100402
[2025-09-03T10:04:02.677+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-09-03T10:04:02.713+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-03T10:04:02.714+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
